applications/RomApplication/custom_strategies/rom_builder_and_solver.h

```diff
@@ -17,6 +17,7 @@
 /* System includes */
 
 /* External includes */
+#include "concurrentqueue/concurrentqueue.h"

```

Comment:
maybe you could use the `AccumReduction` instead

Reply:
Issue is that I need to append multiple items per iteration (x dofs per element/condition).

In another piece of code I append items in some iterations but not in others.

I tried using ReductionUtilities but the code got very ugly very quickly.